scripts/kapplied

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Cool, a script to grab the last applied configuration. I like it

Reply:
at some point soon I'd like to compose this with `ankh template` to extract a single object, convert it to JSON, and then diff agains the last applied configuration.